module/CourseInfo.cpp

```diff
@@ -0,0 +1,83 @@
+/**
+ *@file CourseInfo.cpp
+ *@brief コース情報
+ *@author higuchi/miyashita
+ */
+
+#include "CourseInfo.h"
+using std::string;
+
+// 目標とする輝度と、各区間の名前と距離を設定する
+CourseInfo::CourseInfo(int _brightness)
+  : brightness(_brightness), ranges{ Range("ST_1", 10.0f), Range("CV_1", 5.0f),
+                                     Range("ST_2", 10.0f), Range("CV_2", 5.0f),
+                                     Range("ST_3", 10.0f), Range("CV_3", 5.0f),
+                                     Range("ST_4", 10.0f) }
+{
+  // 入力値が輝度の取りうる範囲外だった場合、128を目標の輝度とする
+  if(_brightness < 0 || 256 <= _brightness) {

```

Comment:
```suggestion
  if(_brightness < 0 || 100 < _brightness) {
```

実は、輝度の範囲は未定なので、とりあえずEV3 APIに合わせて0～100にしておこうか。

以下、未定の理由
EV3 APIにあるgetBrightness関数では、0～100が返される。
一方で、実機の場合はカラーセンサーにちょっと問題があって、今まではgetBrightness関数を使わずに、RGB値から輝度を計算して、0～255の範囲になるようにしていた。
今年は、以下の2点について検証・議論が必要です。
- シミュレータでもカラーセンサーの問題は起きるのか
- 問題が起きるとしても、輝度の範囲はAPIに合わせて0～100にした方が良いのではないか